contracts/STPrime.sol

```diff
@@ -43,6 +43,22 @@ import "./STPrimeConfig.sol";
 contract STPrime is UtilityTokenAbstract, STPrimeConfig {
 	using SafeMath for uint256;
 
+
+    /*
+     *  Storage
+     */
+    /// set when ST' has received TOKENS_MAX tokens;
+    /// when uninitialised minting is not allowed
+    bool private initialized;
```

Comment:
Why `private`?

Reply:
because I felt it should not be exposed / considered for ST' to ever be un-initialised; if ST' is not yet initialised the utility chain is not even worth considering for adding as a Core

Reply:
OK